core/orchestrator.go

```diff
@@ -156,17 +156,11 @@ func (orch *orchestrator) ProcessPayment(payment net.Payment, manifestID Manifes
 		}
 	}
 
+	// Return errors if there were acceptable payment errors but credit is sufficient
```

Comment:
I think this comment is a bit inaccurate because `ProcessPayment()` does not know anything about the credit balance being sufficient/insufficient - it will return an error as long as it encountered *any* error (regardless of whether it was acceptable or not) when invoking `ReceiveTicket()`

Reply:
Leftover note/comment from the discussion we had that I forgot to remove again. I looked over this when doing a final checkup of working trees (initially the logic to check balances was here).

Reply:
Looks like this comment is still here?

Reply:
Weird.. Perhaps pressed the wrong key combo again to save before comitting. Should be fixed now.